web/src/main/resources/static/index.html

```diff
@@ -105,15 +102,13 @@ <h3>Mapping Tools</h3>
           <div class="row">
             <div class="col-md-12">
               <div class="card bg-light mb-3">
-                <div class="card-header">
-                  <h5>Oncotree Version Converter</h5>
-                </div>
                 <div class="card-body">
-                  <p>The OncoTree Version Converter was developed to facilitate forwards- and backwards-mapping of OncoTree codes between OncoTree release versions.</p>
                   <p>
                     <!-- TODO change filename from cross_version_oncotree_translator.py to convert_oncotree_version.py -->
-                    <a class="btn btn-primary btn-sm" href="downloads/cross_version_oncotree_translator.py" role="button" download>Download</a> <a class="btn btn-primary btn-sm" href="http://github.com/cBioPortal/oncotree/blob/master/docs/OncoTree-Version-Converter.md" target="_blank">Documentation</a>
+                    <a class="btn btn-primary btn-sm" href="downloads/cross_version_oncotree_translator.py" role="button" download>Download OncoTree Version Converter Tool</a>
                   </p>
+                  <div id='oncotree-version-converter'>
```

Comment:
one small suggestion - since there isn't anything actually in the `div`, we can make the html element self-ending I believe @mandawilson